ephios/extra/permissions.py

```diff
@@ -26,3 +28,23 @@ def get_groups_with_perms(obj, only_with_perms_in):
         }
     )
     return Group.objects.filter(**group_filters).distinct()
+
+
+class PermissionField(BooleanField):
+    def __init__(self, *args, **kwargs):
+        self.permission_set = kwargs.pop("permissions")
+        super().__init__(*args, **kwargs)
+
+    def set_initial_value(self, user_or_group):
+        self.target = user_or_group
+        self.initial = self.target.permissions.filter(
+            codename__in=map(lambda perm: perm.split(".")[-1], self.permission_set)
+        ).count() == len(self.permission_set)
```

Comment:
This feels a little hacky. Can you please try to

1. Also check for the permission `app_label` if available?
2. Actually compare the sets and not the size of it? I know they aren't of the same types atm. Maybe guardian has a function to help us here.

Reply:
Cannot resolve keyword 'app_label' into field. Choices are: codename, content_type, content_type_id, group, groupobjectpermission, id, name, user, userobjectpermission

Reply:
ah well I wasn't sure :see_no_evil: